R/get_settings_profile.R

```diff
@@ -21,7 +21,7 @@
 #' @param param_space options: real, mulitplier, relative indicates how to interpret the low and high bound values.
 #' real indicates bounds in the parameter space, relative indicates how far to go from the base parameter, and
 #' multiplier indicates that low and high bounds are set at x\% above and below the base parameter.
-#' @param use_prior_like options: Option to include or exclude the prior likelihoods in the likelihood
+#' @param use_prior_like Deprecated Option to include or exclude the prior likelihoods in the likelihood
```

Comment:
I would remove the description of what the deprecated argument used to do, i.e., "Option to ..." and replace it with the text that you have in the PR description regarding this feature is no longer needed and why.